base/db/R/insert.format.vars.R

```diff
@@ -0,0 +1,28 @@
+#' Insert Format and Format-Variable Records 
+#' @param con Bety connection object
+#' @param formats_variables_df A 'data.frame' consisting of entries that correspond to columns in the formats-variables table
+#' @param formats_df A 'data.frame' consisting of entries that correspond to columns in the formats table
+#' @author Liam Burke (liam.burke24@gmail.com)
+#' @return Data frame: Inner join of SQL table and input data frame (as unevaluated "lazy query" table) 
+#' @export
+#' @examples
+#' \dontrun{
+#' bety <- betyConnect
+#'   insert.format.vars(con = bety$con, formats_df, formats_variables_df)
+#' }
+insert.format.vars <- function(con, formats_df, formats_variables_df = NULL){
```

Comment:
All of this is redundant with items in #2012. In the future, it's better to keep PR independent, as this one can't be pulled until that one is finished and then the conflicts between the two are resolved

Reply:
This wasn't supposed to be committed. I agree.